graph_nets/utils_np.py

```diff
@@ -111,11 +111,11 @@ def networkx_to_data_dict(graph_nx,
 
   The networkx graph should be set up such that, for fixed shapes `node_shape`,
    `edge_shape` and `global_shape`:
-    - `graph_nx.nodes(data=True)[i][-1]["features"]` is, for any node index i, a
+    - `graph_nx.nodes(data=True)[i]["features"]` is, for any node index i, a
       tensor of shape `node_shape`, or `None`;
-    - `graph_nx.edges(data=True)[i][-1]["features"]` is, for any edge index i, a
+    - `graph_nx.edges(data=True)[i]["features"]` is, for any edge index i, a
```

Comment:
It seems that in the case of the edges, you actually need the [-1], because graph_nx.edges(data=True) returns a  `OutMultiEdgeDataView` over tuples, each of them with signature: 

(sender_index, receiver_index, OrderedDict([('index', ...), ('features', ...)]))

So the -1 is used to access the dictionary as the last element of the tuple. Also, at least for my version of networkx, the iterable `OutMultiEdgeDataView` does not support indexing, so it requires converting to a list first:
```
list(graph_nx.edges(data=True))[i][-1]["features"]
list(graph_nx.edges(data=True))[i][-1]["index"]
```
Could you please verify if you get the same in your end, and fix accordingly? 

Thanks :)